client/__tests__/util/stateManager/controlsHelpers.test.js

```diff
@@ -1,38 +0,0 @@
-/*
```

Comment:
Why is this file being deleted (or put another way, where are the tests for exported functions from stateManager/controlsHelpers)?

Reply:
The only function being tested in this file was `subsetAndResetGeneLists()` which was deleted in this PR.  Should I just empty the file contents?

Reply:
Best case would be to improve code coverage by adding a few simple tests.   But if not viable, leaving an empty file and a "TODO" would be reasonable.  Totally understand the gap already existed (and was probably my fault!)